frontend/css/playlists.less

```diff
@@ -1,11 +1,60 @@
 @playlistBackgroundColor: #fafafa;
 @descriptionLines: 3;
 
+#playlists-page .playlist-view-options {
+  display: flex;
+  margin-top: 0.8rem;
+  gap: 1rem;
+
+  .playlist-sort-controls {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+}
+
 #playlists-container {
   display: flex;
   flex-wrap: wrap;
   padding-top: 1em;
 
+  &.list-view {
+    flex-direction: column;
+  }
+
+  /* Pure CSS multiline ellipsis by Sagi Shrieber:
+			   http://hackingui.com/a-pure-css-solution-for-multiline-text-truncation/
+			*/
+  .description {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: @descriptionLines * 1.2em;
+    text-align: justify;
+    padding-right: 2em;
+    display: -webkit-box;
+    line-clamp: @descriptionLines;
```

Comment:
The line clamp doesn't combine well with the fact that we have HTML content.
If I use a paragraph, the margin will throw the line height calculation out the window and crop the text awkwardly:
![image](https://github.com/user-attachments/assets/4ebe536c-153e-45d1-8ac7-5c30669ff893)

You can however remedy this with 
```css
#playlists-container .description p {
    margin-bottom: 0;
}
```

![image](https://github.com/user-attachments/assets/f5516a39-62e0-4469-8673-8f091a6656aa)